packages/contracts-bedrock/contracts/test/OptimismMintableERC721.t.sol

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { ERC721 } from "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import { IERC165 } from "@openzeppelin/contracts/utils/introspection/IERC165.sol";
+import { Strings } from "@openzeppelin/contracts/utils/Strings.sol";
+import { ERC721Bridge_Initializer } from "./CommonTest.t.sol";
+import { OptimismMintableERC721 } from "../universal/OptimismMintableERC721.sol";
+
+contract OptimismMintableERC721_Test is ERC721Bridge_Initializer {
+    ERC721 internal L1Token;
+    OptimismMintableERC721 internal L2Token;
+
+    event Transfer(address indexed from, address indexed to, uint256 indexed tokenId);
+
+    event Mint(address indexed account, uint256 tokenId);
+
+    event Burn(address indexed account, uint256 tokenId);
+
+    function setUp() public override {
+        super.setUp();
+
+        // Set up the token pair.
+        L1Token = new ERC721("L1Token", "L1T");
+        L2Token = new OptimismMintableERC721(
+            address(L2Bridge),
+            1,
+            address(L1Token),
+            "L2Token",
+            "L2T"
+        );
+
+        // Label the addresses for nice traces.
+        vm.label(address(L2Token), "L2Token");
```

Comment:
Not sure if we use the same label for the erc20 L1/L2 tokens, might want to be more specific `L1ERC721Token`